File: benfica-frontend/frontend/src/app/cliente/shared/entidade/clienteRetornoCompletoDTO.ts
import { Endereco } from "src/app/usuario/shared/entidade/endereco";
import { Seguro } from "./seguro";
import { ContatoCliente } from "./contatoCliente";

export class ClienteRetornoCompletoDTO {

  constructor(){
    this.id = 0;
    this.nome = '';
    this.cpf = '';
    this.endereco = new Endereco();
    this.seguro = [];
    this.contatoCliente = [];
    this.tipoSeguro = 0;
    this.objetoSeguro = new Object();
  }

  id: number;
  nome: string;
  cpf: string;
  seguro: Seguro[];
  contatoCliente: ContatoCliente[];
  endereco: Endereco;
  tipoSeguro: number;
  objetoSeguro: object;
}

File: benfica-api/backend/src/main/java/com/benfica/dto/ClienteRetornoCompletoDTO.java
package com.benfica.dto;

import java.util.List;

import com.benfica.model.ContatoCliente;
import com.benfica.model.Endereco;
import com.benfica.model.Seguro;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ClienteRetornoCompletoDTO {

	private Long id;
	private String nome;
	private String cpf;
	private List<ContatoCliente> contatoCliente;
	private List<Seguro> seguro;
	private Long tipoSeguro;
	private Endereco endereco;
	private List<Object> objetoSeguro;
}
